package.json

```
@@ -37,6 +37,7 @@
     "develop": "gatsby develop --prefix-paths",
     "lint": "./node_modules/.bin/eslint src/ tests/",
     "lintfix": "prettier-eslint --write \"src/**/*.js\" \"tests/**/*.js\"",
```

Since there's a lot of JS code in `preview/` now, we can turn on Prettier in that directory:
```suggestion
    "lintfix": "prettier-eslint --write \"src/**/*.js\" \"tests/**/*.js\" \"preview/**/*.js\"",
```